en/streaming-integrator/docs/setup/upgrading-from-previous-version.md

```diff
@@ -0,0 +1,50 @@
+# Upgrading from Streaming Integrator 1.0.1
```

Comment:
This should be 1.0.0 right? we don't have a 1.0.1 release for SI

Reply:
Corrected.